cmd/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+var errorPlatformNotSupported error = errors.New("platform not supported, only supporting windows and drawin")
+
+func IsInstalled() (bool, error) {
```

Comment:
It would be best if these methods (IsInstalled, Install, Run) are defined on the struct.

Reply:
Fixed in c846d7a44aa2fdf39f5b0a44b37d9a05921da928.